### ISSUE TITLE
no connector for micro USB

### ISSUE THREAD

**Issue by korol-sas** [CrealityOfficial/K1_Series_Annex#13]
https://github.com/CrealityOfficial/K1_Series_Annex/releases/tag/V1.0.0 Hello, today when updating my K1 to version V1.3.3.5 an error occurred, after rebooting the printer does not start, I guess there is no free space in the memory, I cannot perform the actions according to this instruction, because there is no connector on my motherboard for micro USB

**Comment by e2s-consult** [comment]
same here

**Comment by Hammerfest** [comment]
Either of you have a picture of your mainboard?
Interested to see differences

**Comment by ozzik555** [comment]
That's okay, I have mine connector soldered, but doesn't work 😃

**Comment by korol-sas** [comment]
> That's okay, I have mine connector soldered, but doesn't work 😃

Yes, I also soldered, it does not work, there are still many components missing on the board

**Comment by KITREKSU** [comment]
> > That's okay, I have mine connector soldered, but doesn't work 😃
> 
> Yes, I also soldered, it does not work, there are still many components missing on the board

Brother , I too soldered micro usb port , no use. Is there any other method to fix this ? Anu other port which support manual powering up ?

**Comment by korol-sas** [comment]
> > > That's okay, I have mine connector soldered, but doesn't work 😃
> > 
> > 
> > Yes, I also soldered, it does not work, there are still many components missing on the board
> 
> Brother , I too soldered micro usb port , no use. Is there any other method to fix this ? Anu other port which support manual powering up ?

I abandoned this idea and just bought a new mainboard pcb and replaced it.

**Comment by KITREKSU** [comment]
> > > > That's okay, I have mine connector soldered, but doesn't work 😃
> > > 
> > > 
> > > Yes, I also soldered, it does not work, there are still many components missing on the board
> > 
> > 
> > Brother , I too soldered micro usb port , no use. Is there any other method to fix this ? Anu other port which support manual powering up ?
> 
> I abandoned this idea and just bought a new mainboard pcb and replaced it.

I fixed it brother , you either solder a micro usb port or a usb data cable with +ve -ve d+ and d - directly into the board , then connect the usb with your pc .. pc may not detect the usb, reason +ve supply line to the board is incomplete.... *Just plug in the power cord to the main and turn on the printer* then the usb will detect by the pc . And then install the driver and flash the firmware. Simple

**Comment by gebauer** [comment]
Hi @KITREKSU,

I have the same problem. I already found out that the pins directyl next to the microusb are directly connected, so this makes soldering for me much easier. Did I understand you correctly, I only connect - / + to 24V and the baord should be recognised?

<img width="2596" height="1822" alt="Image" src="https://github.com/user-attachments/assets/7a7ad1ba-2834-4427-8527-33917d4f2cec" />

Thanks,
Jan
